Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/PrototypeClassK.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Neos\Flow\Tests\Functional\ObjectManagement\Fixtures;
+
+use Neos\Flow\Annotations as Flow;
+
+/**
+ * A class to test static compile functionality
+ */
+class PrototypeClassK
+{
+    public function getMicrotime(): float
+    {
+        return static::compiledStaticallyMethod();
+    }
+
+    /**
+     * Method that should get static compiled into the proxy, saving some processing power in production,
+     * but also providing the exact same result on every call.
+     *
+     * @return float
+     * @Flow\CompileStatic
+     */
+    public static function compiledStaticallyMethod(): float
+    {
+        return microtime(true);
```

Comment:
might be better to use random string instead as if your mashine is fast enough it might yield the same result either way?

Reply:
I guess in theory? How long of a random string do we want to use?